app/views/users/show.html.erb

```diff
@@ -1,10 +1,34 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-  </head>
+<div class='main-div'>
+  <div class="show-container"> 
+    <img src="<%= @user.photo %>" alt="Profile photo" />
+    <div>
+      <p><%= @user.name %></p>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+</div>
 
```

Comment:
- [ ] I'm getting an error accessing this route:

![image](https://github.com/techmoves/movesBloc/assets/77069011/7d4706d8-b99c-4901-8761-6460a11da05d)

This is because you don't have any `@user` variable in your controller. Please consider fixing this! 🤓 🔧